source/material_model/latent_heat.cc

```diff
@@ -177,11 +177,11 @@ namespace aspect
               for (unsigned int phase=0; phase<phase_function.n_phase_transitions(); ++phase)
                 {
                   const double depth = this->get_geometry_model().depth(in.position[i]);
```

Comment:
Since you changed this to `position` below, you might as well change it here too.